assets/src/dashboard/app/reducer/stories.js

```diff
@@ -120,7 +120,17 @@ function storyReducer(state, action) {
       };
 
     case ACTION_TYPES.FETCH_STORIES_SUCCESS: {
-      const fetchedStoriesById = action.payload.stories.map(({ id }) => id);
+      const fetchedStoriesById = [];
+      const reshapedStories = action.payload.stories.reduce((acc, current) => {
+        if (!current) {
+          return acc;
+        }
+        fetchedStoriesById.push(current.id);
+        acc[current.id] = reshapeStoryObject(action.payload.editStoryURL)(
```

Comment:
dig the move to reshape in the reducer so that all of our shaping happens in 1 location